### PR TITLE
[ACTION] AWeber: Create or Update subscriber #5936

### PR DESCRIPTION
**Pull request by lcaresia**
### <samp>🤖 Generated by Copilot at d45a0d4</samp>

This pull request adds a new action component to the Aweber source that enables creating or updating subscribers in a list. It also contains a potential versioning issue in the `package.json` file that needs to be fixed.

### <samp>🤖 Generated by Copilot at d45a0d4</samp>

> _`Aweber` source grows_
> _New action to update list_
> _Version number wrong_

### <samp>🤖 Generated by Copilot at d45a0d4</samp>

*  Add a new action component to create or update a subscriber in Aweber ([link](https://github.com/PipedreamHQ/pipedream/pull/5937/files?diff=unified&w=0#diff-9a779debf8dc1e8c2e7961d201c40ec1cd434c2cd65dc30c5ba6e1a09eaca0d8R1-R88))
* Fix the version number of the Aweber source package ([link](https://github.com/PipedreamHQ/pipedream/pull/5937/files?diff=unified&w=0#diff-91b581228c58f60cc3d4d7275659e9d3e7b830289946d3fc80ea86351a22d92aL3-R3))

### CODE REVIEW

File: components/aweber/actions/create-or-update-subscriber/create-or-update-subscriber.mjs [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import aweberApp from "../../aweber.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "aweber-create-or-update-subscriber",
+  name: "Create Or Update Subscriber",
+  description: "Create subscriber if the subscriber email is not existing or update the information for the specified subscriber by email. [See the docs here](https://api.aweber.com/#tag/Subscribers/paths/~1accounts~1{accountId}~1lists~1{listId}~1subscribers/patch).",
+  type: "action",
+  version: "0.0.2",
```
Comment (andrewjschuang) [inline components/aweber/actions/create-or-update-subscriber/create-or-update-subscriber.mjs:9]:
```suggestion
  version: "0.0.1",
```

**Review by andrewjschuang — COMMENTED** [review]
Thanks!

Question: What happens if the subscriber doesn't exist? From what I'm looking here, this action only tries to update the subscriber.

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[AWeber_5937_2221.pdf](https://github.com/PipedreamHQ/pipedream/files/11172042/AWeber_5937_2221.pdf)

File: components/aweber/actions/create-or-update-subscriber/create-or-update-subscriber.mjs [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import aweberApp from "../../aweber.app.mjs";
+
+export default {
+  key: "aweber-create-or-update-subscriber",
+  name: "Create Or Update Subscriber",
+  description: "Create subscriber if the subscriber email is not existing or update the information for the specified subscriber by email. [See the docs here](https://api.aweber.com/#tag/Subscribers/paths/~1accounts~1{accountId}~1lists~1{listId}~1subscribers/patch).",
+  type: "action",
+  version: "0.0.5",
```
Comment (vellames) [inline components/aweber/actions/create-or-update-subscriber/create-or-update-subscriber.mjs:8]: `0.0.1`

**Review by vellames — CHANGES_REQUESTED** [review]
Hi there. Just a minor

**Review by vellames — APPROVED** [review]

**Comment by vunguyenhung** [comment]
Hello @lcaresia, in the last report, I meant only the Name should be updated means that "When I set the Name prop, only the Name should be changed, the Tag and Notes prop when not set is kept as is". Could you add back the Tags and the Notes and follow this behavior?

I'm sorry for the miscommunication 🙏

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[AWeber_5937_2214.pdf](https://github.com/PipedreamHQ/pipedream/files/11212967/AWeber_5937_2214.pdf)

**Review by vellames — APPROVED** [review]

**Pull request merged**